README.md

```diff
@@ -10,28 +10,186 @@ The features are available before an official release so that users and collabor
+    ```
+    </details>
+
+- efficient<sup>(2)</sup> and generic<sup>(1)</sup> [replay buffers](torchrl/data/replay_buffers/replay_buffers.py) that with modularized storage:
+    <details>
+      <summary>Code</summary>
+    
+    ```python
+    storage = LazyMemmapStorage(  # memory-mapped (physical) storage
+        cfg.buffer_size,
+        scratch_dir="/tmp/"
+    )
+    buffer = TensorDictPrioritizedReplayBuffer(
+        buffer_size=10000,
+        alpha=0.7,
+        beta=0.5,
+        collate_fn=lambda x: x,
+        pin_memory=device != torch.device("cpu"),
+        prefetch=10,  # multi-threaded sampling
+        storage=storage
+    )
+    ```
+    </details>
+
+- [interfaces for environments](torchrl/envs)
+from common libraries (OpenAI gym, deepmind control lab, etc.)<sup>(1)</sup> and [wrappers](torchrl/envs/vec_env.py) for parallel execution<sup>(2)</sup>, 
+as well as a new pytorch-first class of [tensor-specification class](torchrl/data/tensor_specs.py):
+    <details>
+      <summary>Code</summary>
+    
+    ```python
+    env_make = lambda: GymEnv("Pendulum-v1", from_pixels=True)
+    env_parallel = ParallelEnv(4, env_make)  # creates 4 envs in parallel
+    tensordict = env_parallel.rollout(max_steps=20)
+    assert tensordict.shape == [4, 20]  # 4 envs, 20 steps rollout
+    ```
+    </details>
+
+- cross-library [environment transforms](torchrl/envs/transforms/transforms.py)<sup>(1)</sup>, 
+executed on device and in a vectorized fashion<sup>(2)</sup>, 
+which process and prepare the data coming out of the environments to be used by the agent:
+    <details>
+      <summary>Code</summary>
+    
+    ```python
+    env_make = lambda: GymEnv("Pendulum-v1", from_pixels=True)
+    env_base = ParallelEnv(4, env_make, device="cuda:0")  # creates 4 envs in parallel
+    env = TransformedEnv(
+        env_base, 
+        Compose(ToTensorImage(), ObservationNorm(loc=0.5, scale=1.0)),  # executes the transforms once and on device
+    )
+    tensordict = env.reset()
+    assert tensordict.device == torch.device("cuda:0")
+    ```
+    </details>
+
+- various tools for distributed learning (e.g. [memory mapped tensors](torchrl/data/tensordict/memmap.py))<sup>(2)</sup>;
+- various [architectures](torchrl/modules/models/) and models (e.g. [actor-critic](torchrl/modules/tensordict_module/actors.py))<sup>(1)</sup>:
+    <details>
+      <summary>Code</summary>
+    
+    ```python
+    common_module = ConvNet(
+        bias_last_layer=True,
+        depth=None,
+        num_cells=[32, 64, 64],
+        kernel_sizes=[8, 4, 3],
+        strides=[4, 2, 1],
+    )
+    common_module = TensorDictModule(
+        common_module,
+        in_keys=["pixels"],
+        out_keys=["hidden"],
+    )
+    policy_module = NormalParamsWrapper(
+        MLP(
+            num_cells=[64, 64],
+            out_features=32,
+            activation=nn.ELU,
+        )
+    )
+    policy_module = ProbabilisticTensorDict(  # stochastic policy
```

Comment:
This could be worth elaborating on? So it seems like a tensor dict can take as a parameter another tensordict which helps with your modularity story

Reply:
`ProbabilisticTensorDict` should read `ProbabilisticTensorDictModule`